modules/skybrowser/shaders/target_fs.glsl

```diff
@@ -66,6 +66,13 @@ float createCrosshair(in float linewidth, in float ratio, in vec2 coord) {
   return crosshairHorizontal + crosshairVertical;
 }
 
+float roundedRectangle(vec2 coord, vec2 size, vec4 radius) {
```

Comment:
This function is a bit magic to me (mainly the 4-dim radius and the fact that the function overwrites the input values that makes it difficult to understand)

Maybe add a short description of how it works or some supporting comment in the function? Maybe unnecessary, I don't know. Open to opinions

Reply:
Yeah tbh I'm not really sure how it works either, I found the formula online hehe. I removed the 4 dim radius since we don't use that anyways and wrote a comment. Can't see that the function overwrites the input values though